docs/source/torch.ao.ns._numeric_suite_fx.md

```diff
@@ -0,0 +1,30 @@
+(torch_ao_ns_numeric_suite_fx)=
+
+# `torch.ao.ns._numeric_suite_fx`
+
+```{warning}
+This module is an early prototype and is subject to change.
+```
+
+:::{module} torch.ao.ns._numeric_suite_fx
+:members:
+:member-order: bysource
+:::
```

Comment:
this needs to be original from the .rst file wrapped in the {eval-rst} directive